### PR TITLE
Add three new functions to common.py

### PR DESCRIPTION
**Pull request by dp247**
A few more functions for common.py - these may be useful for calendars where we know the collection day, but have no way of scraping data and have to use our own functions to generate them.

## is_weekend()
A counterpart to is_holiday(), provides an easy way of finding out if a date is on a Saturday/Sunday.

## is_working_day()
I'm defining 'working day' as any day Monday to Friday that is not a holiday. This function is a wrapper for both is_holiday and is_weekend - if one is True for the given date, false is returned.

## get_next_working_day()
Could be useful when used with the above - returns the next working day according to the criteria above

### CODE REVIEW

**Comment by dp247** [comment]
Feel free to ignore the revert, I wasn't looking at the tickboxes in Pycharm LOL

**Comment by dp247** [comment]
Also happy one thousandth issue!

File: uk_bin_collection/uk_bin_collection/common.py [hunk 1]
```diff
@@ -162,6 +161,31 @@ def is_holiday(date_to_check: datetime, region: Region = Region.ENG) -> bool:
         return False
 
 
+def is_weekend(date_to_check: datetime) -> bool:
+    """
+    Checks if a given date is a weekend
+    :param date_to_check: Date to check if it falls on a weekend
+    :return: Bool - true if a weekend day, false if not
+    """
+    return True if date_to_check.date().weekday() >= 5 else False
```
Comment (dannya) [inline uk_bin_collection/uk_bin_collection/common.py:170]: @dp247 couldn't this just be:

`return date_to_check.date().weekday() >= 5`

?

Comment (dp247) [reply uk_bin_collection/uk_bin_collection/common.py:170]: For sure.

**Review by robbrad — CHANGES_REQUESTED** [review]
Looks to be some extra tabs added to the common functions @dp247 ? Do the unit tests work?

Can you add the coverage for the new tests?

**Comment by dp247** [comment]
> Looks to be some extra tabs added to the common functions @dp247 ? Do the unit tests work?
> 
> Can you add the coverage for the new tests?

Yes, I'll get this sorted when I've returned from my travels

**Review by robbrad — CHANGES_REQUESTED** [review]
I cloned this and it seams the common unit tests are no longer executing - I can review further on Sunday night

**Review by robbrad — APPROVED** [review]

**Pull request merged**